def.bzl

```diff
@@ -57,6 +57,8 @@ def _gazelle_runner_impl(ctx):
         args.extend(["-build_tags", ",".join(ctx.attr.build_tags)])
     args.extend([ctx.expand_location(arg, ctx.attr.data) for arg in ctx.attr.extra_args])
 
+    env = "\n".join(["export %s='%s'" % (x, y) for (x, y) in ctx.attr.env.items()])
```

Comment:
using `export VAR=VALUE` means that these values will remain in the user's shell after a Bazel's run?

Should we simply do `VAR=VALUE` instead?

Reply:
In *nix you can't modify environment variables from the caller, you can only modify on the current process and it's childs, even on osx you can't not sure about Windows

Reply:
@sluongng are you satisfied with my explanation? Can we resolve this thread?

Reply:
Yup, sounds good to me. 👍